### PR TITLE
update: github workflows and documentation

### PR DESCRIPTION
**Pull request by SanGeeky**
## Objective
Update Github Workflows related to Continuous Integration and publishing the package to npm. Furthermore, updates the documentation pertaining to LICENSE and CONTRIBUTING.

This PR improves the repository before publishing the initial release of the project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ We want to make it as easy as possible to contribute changes that help the [tick
 * Fork the repository on GitHub.
 * Try to keep your local repository in a "rebased" state.
 * Set the project up.
-  * Install the dependencies running `yarn`.
-  * To create a build in development mode, run `yarn build:dev`.
-  * To create a production bundle, run `yarn build`.
-  * Run tests with `yarn test`.
+  * Install the dependencies running `npm run`.
+  * To create a build in development mode, run `npm run build:dev`.
+  * To create a production bundle, run `npm run build`.
```
Comment (miguelnietoa) [inline CONTRIBUTING.md:17]: These commands do not exist under "scripts" inside the `package.json`. Do you consider they're required? 🤔

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Of course, feel free to create a new issue if you think something needs to be ad
   * This is usually the `main|master` branch.
   * Please avoid working directly on the `main|master` branch.
```
Comment (miguelnietoa) [inline CONTRIBUTING.md:34]: Let's remove the `master` branch
```suggestion
  * This is usually the `main` branch.
  * Please avoid working directly on the `main` branch.
```

**Review by miguelnietoa — CHANGES_REQUESTED** [review]
Hi there @SanGeeky, some changes are required!

File: CONTRIBUTING.md (hunk 1 above)
Comment (SanGeeky) [reply CONTRIBUTING.md]: Not for now, maybe could be an improvement in the future.

**Review by miguelnietoa — CHANGES_REQUESTED** [review]
Please remove `master` from lines 31 and 44 in the `CONTRIBUTING.md` file, and that's it!

**Review by miguelnietoa — APPROVED** [review]

**Pull request merged**